docassemble/AssemblyLine/language.py

```diff
@@ -13,6 +14,32 @@
 ]
 
 
+def _package_name(package_name: Optional[str] = None):
+    """Get package name without the name of the current module"""
```

Comment:
```suggestion
    """Get package name without the name of the given module. By default this is `docassemble.AssemblyLine.language`"""
```

If I understand this correctly (that `__name__` should always be the `language.py` module). "Current module" is just vague enough that we should avoid it IMO